config/user.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "id": 3003,
+        "token":"45erkjherht45495783",
+        "username":"mercy chisom",
+        "password" :"uzodhhhd2ei"
+    },
+    {
+        "id": 2560,
+        "token":"45erkjherht45495783",
+        "username":"daisy mike",
+        "password" :"mikeijisda"
+    },
+    {
+        "id": 15601,
+        "token":"45erkjherht45495783",
+        "username":"victorhart mike",
+        "password" :"killjs123"
+    }
+]
```

Comment:
Create a data or mocks folder and move this file there

Reply:
have corrected the file. already in a mock folder